task_api/app/controllers/api/labels_controller.rb

```diff
@@ -34,7 +35,15 @@ def bulk_create
 
   private
 
+  def validate_task_exists
+    if !Task.exists?(id: params[:task_id])
+      render json: { message: 'Specified Task Not Found', data: nil }, status: :not_found
+    end
```

Comment:
`if の否定` は `unless` の方が良いですね。
if 文の中が1行なら、後置if/unless の方が見やすいかも。

```suggestion
    render json: { message: 'Specified Task Not Found', data: nil }, status: :not_found unless Task.exists?(id: params[:task_id])
```